packages/loopring_v3/contracts/aux/bridge/IBatchDepositor.sol

```diff
@@ -14,6 +14,7 @@ interface IBatchDepositor
         address owner;
         address token;
         uint96  amount;
+        uint16  tokenID;
```

Comment:
I added this optional field to get rid of `BatchDeposit` assuming this new field won't increase the data layout. Let me know if this works.

Reply:
I wouldn't do this because it can be very confusing, especially because it's part of the external interface. We already had to be very careful that the data passed in matches the correct values, because it's not because some `token` and `tokenID` is passed in from somewhere that `token == getTokenAddress(tokenID)`. So now in the code we are in the situation where:
- `token` is the only value we can use, we have to add comments when this is the case and follow them strictly
- `tokenID` is the only value we can use, we have to add comments when this is the case and follow them strictly
- we cannot depend on `token == getTokenAddress(tokenID)` because data is often just passed in from somewhere else, so they won't match

It seems this is quite error prone even for us in the Bridge. An extra struct only we normally use internally seems to be the lesser of two evils. It will also slightly increase gas costs.

Reply:
reverted